_backend/email/os-payment.php

```diff
@@ -8,17 +8,19 @@
 require_once __DIR__ . '/../../_backend/preload.php';
 require_once __DIR__ . '/../../_backend/config.loader.php';
 
-$mandrill = new Mandrill($config['mandrill_key']);
+$mailchimp = new MailchimpTransactional\ApiClient();
+$mailchimp->setApiKey($config['mandrill_key']);
```

Comment:
Is there a reason why you did not create a new key here or just rename the existing key? Its might be hard to remember in the future why the MailChimp API is consuming the Mandril key